ui/packages/catalog-ui-search/src/main/webapp/js/model/Workspace.js

```diff
@@ -136,14 +136,17 @@ module.exports = PartialAssociatedModel.extend({
         .length === 0
     ) {
       this.set('saved', false)
+      this.set('metacard.modified', Date.now())
```

Comment:
✏️ I recommend encapsulating this in an instance method and calling it at the various places that modification stamp is required:
```
updateMetacardModifiedTimestamp() {
   this.set('metacard.modified', Date.now())
}
```